capstone-frontend/src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { TestApi } from "./components/TestApi";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div id="testApiID">
+      <TestApi />
```

Comment:
Also not sure this should necessarily be checked in. If you need to test your API, try to do so with unit tests. It's probably OK to check in a bare-bones front-end component that basically just dumps the API response onto the page for now, but try to avoid things that are just for validating that other pieces work and are meant to be replaced entirely. If I'm misunderstanding please reply here or add comments to the source code.